cosmwasm/ibc-union/lightclient/ethereum/src/client.rs

```diff
@@ -96,17 +99,41 @@ impl IbcClient for EthereumLightClient {
     }
 
     fn verify_creation(
-        _client_state: &Self::ClientState,
+        client_state: &Self::ClientState,
         _consensus_state: &Self::ConsensusState,
-    ) -> Result<Option<Vec<VerifyCreationResponseEvent>>, IbcClientError<EthereumLightClient>> {
-        Ok(None)
+    ) -> Result<ClientCreation<Self>, IbcClientError<Self>> {
+        let Some(initial_sync_committee) = client_state.initial_sync_committee.as_ref() else {
+            return Err(Error::NoInitialSyncCommittee.into());
+        };
+        let mut client_state = client_state.clone();
+        // We only require this at the creation phase. The client then manages the committees in a separate storage.
+        client_state.initial_sync_committee = None;
```

Comment:
something we can look into perhaps is an additional "client bootstrap" state, only provided during client creation?

Reply:
I thought about this and initially wanted to have that, but then chose this path because it would effect the public IBC api for a specific implementation.

Reply:
ref: https://github.com/unionlabs/union/issues/3921